packages/dropdown/src/HighlightContext/HighlightProvider.types.tsx

```diff
@@ -0,0 +1,7 @@
+import { HighlightBehavior } from '../Dropdown/Dropdown.types';
+
+export interface HighlightProvider {
+  highlightedRef?: HTMLElement | null;
+  setHighlightedRef?: (ref: HTMLElement | null) => void;
```

Comment:
If these are `elements` they should be named as such (not `ref`)

Reply:
Done